lib/init.coffee

```diff
@@ -78,127 +81,76 @@ module.exports =
       helpers = require 'atom-linter'
       voucher = require 'voucher'
       fs = require 'fs'
-      if @verboseLogging
-        @_log 'requiring modules finished.'
+      @_log 'requiring modules finished.'
 
-    if @verboseLogging
-      @_log 'providing linter, examining javac-callability.'
+    @_log 'providing linter, examining javac-callability.'
 
     grammarScopes: ['source.java']
     scope: 'project'
     lintOnFly: false       # Only lint on save
+
     lint: (textEditor) =>
       filePath = textEditor.getPath()
       wd = path.dirname filePath
       searchDir = @getProjectRootDir() || path.dirname filePath
+
       # Classpath
       cp = ''
 
-      if @verboseLogging
-        @_log 'starting to lint.'
+      # Sourcepath - this is a fair default for many Java projects
+      sp = 'src/main/java/'
```

Comment:
I really dislike hardcoded defaults. How would it be, if we check if the config-setting if it references to a file or a directory?